### ISSUE TITLE
Issue to run "python forward_run.py"

### ISSUE THREAD

**Issue by flydream0428** [jtwhite79/pyemu#104]
I am using miniconda to manage my Python packages. Although I have added Python path into the system environment path variable. I still cannot run "python forward_run.py" directly. It will raise the following error if I run it directly from a cmd window. However, if I run `activate base` first, it works well. I have tried to add  `activate base`  into the PEST control file. PESTPP cannot start activate command. Any suggestion? 

Traceback (most recent call last):
  File "forward_run.py", line 3, in <module>
    import numpy as np
  File "C:\Users\cui00e\AppData\Local\Continuum\miniconda3\lib\site-packages\numpy\__init__.py", line 140, in <module>
    from . import _distributor_init
  File "C:\Users\cui00e\AppData\Local\Continuum\miniconda3\lib\site-packages\numpy\_distributor_init.py", line 34, in <module>
    from . import _mklinit
ImportError: DLL load failed: The specified module could not be found.

**Comment by jtwhite79** [comment]
are you starting PESTPP from the anaconda prompt (with the `activate base`)?

**Comment by flydream0428** [comment]
Thanks, J. I guess this is an issue related to the recent Anaconda distribution. I have found a workaround. I created a batch file as follows. It works now. 
====
call C:\Users\cui00e\AppData\Local\Continuum\miniconda3\Scripts\activate.bat
python forward_run.py
mod2obs <mod2obs.in
========

**Comment by briochh** [comment]
@flydream0428, yes you are defo experiencing an issue with your anaconda distribution. Recent versions rely on environments being activated, when not activated this can result in packages not being found (at least on windows). When running from anaconda prompt this activation is done automatically. I believe this is to try and prevent conda versions of python from stomping on system versions. You’ve probably have found heaps of info on this already - the solution is (or has been) messy (I don’t think I have solved it the same way more than once). 
From memory, things I have had luck with:
0. Registering the conda version as my default python on install (check box on install). This adds a few items to your system path but I’d be guessing right now as to what they are.
1. I think there is an anaconda init command (or something like that sets up a few batch scripts). And possibly registers the activation with the cmd (or powershell) etc etc. initialisation.
2. The other thing that has caught me out is having appropriate access to these batch scripts - so may be worth checking permissions where these conda batch scripts are stored.
Your work around is essentially doing the activation and run in two steps. I think this is a fine approach. The good people at continuum anaconda would rather we manually activated environments anyway before running python (I think). In this vein 

```
conda activate base
python forward_run.py
```
Should work. This could be placed in a separate batch script that PEST runs. Replace “base” with whatever env. you want to run in.

Hope this helps
